package.json

```diff
@@ -663,10 +663,6 @@
 								"type": "string",
 								"description": "Workspace root."
 							},
-							"packages": {
```

Comment:
This should still be an option, since a Dart process can be launched with an arbitrary packages file rather than whatever would be found with the default logic relative to the entry point. (That's where I hit a wall on trying to get this working with the observatory: the root library path works for guessing, but I couldn't find anywhere that explicitly had either the packages file or the full paths for each library)

Reply:
Ah ok, that makes sense :)

If we put this back and the user doesn't specify it, should we prefer the one in their current workspace, or from the VM? My concern with the one from the workspace is that if they have the wrong project open it'll just be wrong.

Reply:
Guessing based on the VM sounds reasonable to me (assuming the assumptions about isolate ordering are valid); I can't think of any reason the one in the folder would be more likely to be right if a non-default packages file was used to launch the process.